data/chaindata.json

```diff
@@ -3442,13 +3450,69 @@
             "symbol": "EURC.s",
             "decimals": 12,
             "ed": "1000",
-            "onChainId": "{\"type\":\"Stellar\",\"value\":{\"type\":\"AlphaNum4\",\"value\":{\"code\":\"bin:EURC\",\"issuer\":\"hex:0x2112ee863867e4e219fe254c0918b00bc9ea400775bfc3ab4430971ce505877c\"}}}"
+            "onChainId": "{\"type\":\"Stellar\",\"value\":{\"type\":\"AlphaNum4\",\"value\":{\"code\":\"bin:EURC\",\"issuer\":\"hex:0x2112ee863867e4e219fe254c0918b00bc9ea400775bfc3ab4430971ce505877c\"}}}",
+            "coingeckoId": "eurc"
+          },
+          {
+            "symbol": "EURC.s",
+            "decimals": 12,
+            "ed": "1000",
+            "onChainId": "{\"type\":\"Stellar\",\"value\":{\"type\":\"AlphaNum4\",\"value\":{\"code\":\"bin:EURC\",\"issuer\":\"hex:0xcf4f5a26e2090bb3adcf02c7a9d73dbfe6659cc690461475b86437fa49c71136\"}}}",
+            "coingeckoId": "eurc"
           },
           {
             "symbol": "NGNC.s",
             "decimals": 12,
             "ed": "1000",
             "onChainId": "{\"type\":\"Stellar\",\"value\":{\"type\":\"AlphaNum4\",\"value\":{\"code\":\"bin:NGNC\",\"issuer\":\"hex:0x241afadf31883f79972545fc64f3b5b0c95704c6fb4917474e42b0057841606b\"}}}"
+          },
+          {
+            "symbol": "BRZ",
+            "decimals": 18,
+            "ed": "1000",
+            "onChainId": "{\"type\":\"XCM\",\"value\":4}",
+            "coingeckoId": "brazilian-digital-token"
+          },
+          {
+            "symbol": "PINK",
+            "decimals": 10,
+            "ed": "1000",
+            "onChainId": "{\"type\":\"XCM\",\"value\":7}"
+          },
+          {
+            "symbol": "HDX",
+            "decimals": 12,
+            "ed": "1000",
+            "onChainId": "{\"type\":\"XCM\",\"value\":8}",
+            "coingeckoId": "hydration"
+          },
+          {
+            "symbol": "ASTR",
+            "decimals": 18,
+            "ed": "1000",
+            "onChainId": "{\"type\":\"XCM\",\"value\":9}",
+            "coingeckoId": "astar"
+          },
+          {
+            "symbol": "vDOT",
+            "decimals": 10,
+            "ed": "1000000",
+            "onChainId": "{\"type\":\"XCM\",\"value\":10}",
+            "coingeckoId": "voucher-dot"
+          },
+          {
+            "symbol": "BNC",
+            "decimals": 12,
+            "ed": "10000000000",
```

Comment:
Maybe I misunderstand how this definition works but [the ED of all our non-native assets is 1000](https://github.com/pendulum-chain/pendulum/blob/c65378960579bb04ebe99466e79f47838fc29c48/runtime/pendulum/src/lib.rs#L819-L827). This also applied to `vDOT`.

Reply:
True, but I think this differs when you register non-native assets from other chains. They usually tell us the existential deposit of their asset that we should use for registration. And that's why some tokens have a non-`1000` ED also in our asset registry metadata. And this change would just reflect that.

Reply:
Hmm, but this is then just a definition in the registry, not the actual ED. The actual ED is only determined by orml-tokens and there it is 1000 for each asset. Should we change this then in order to have consistent definitions? It probably doesn't matter too much but could be confusing otherwise.

Reply:
It may be better to consult what exactly are they using this ED for, if it is only for receiving or also sending, or something else. We can easily open a new PR to double check all our registered assets once we know this.

Reply:
I would assume that the ED is used by whatever tokens pallet they use and is not related to sending, receiving, etc. The pallet will just make sure that – whatever happens to the balance through whatever operation – it will either be at least the ED or 0.

Reply:
@ebma I am not even sure whether this is sufficient. Even if we set our ED to `10000000000`, as long as we don't control that the minimum sending amount is also `10000000000`, then anyone who owns `10000000001` tokens on Pendulum could just send `1` token via XCM to an empty target account and that target account would not reach the required ED. And then again, even we define in our XCM configuration that the minimum transfer amount is at least the ED, then this might be overly restrictive if sending an amount to an account that already has more than the ED on the target chain.

Reply:
> Even if we set our ED to 10000000000, as long as we don't control that the minimum sending amount is also 10000000000, then anyone who owns 10000000001 tokens on Pendulum could just send 1 token via XCM to an empty target account and that target account would not reach the required ED.

True, but I guess the wallet application would use the information about the existential deposit to prevent the user from doing exactly that.

> And then again, even we define in our XCM configuration that the minimum transfer amount is at least the ED, then this might be overly restrictive if sending an amount to an account that already has more than the ED on the target chain.

I think this is the bigger problem. The wallet application could also add extra handling for this though. The wallet can check the balance of the destination account on the other chain and then decide whether the amount that the user is trying to send makes sense or not. I don't know whether Talisman or the other wallet apps are doing this. As @gianfra-t said, we would need to ask the devs of Talisman in order to confirm. 

Now you could argue that if the wallet app integrates extra handling to check the existential deposit before transferring (as I suggested in my previous point), then we could also continue using our custom existential deposit that is potentially different from the one on the source chain as it shouldn't matter. As long as we don't know if/how wallets handle the existential deposit, it's difficult to make an informed decision.

Reply:
I would be very impressed if wallets do this test. It is probably easy to check ourselves whether this is the case or not.

Reply:
@pendulum-chain/product do we have a communication channel with the Talisman team on Telegram?

Reply:
yes we have @ebma